openstack/placement/templates/stamp-alembic-job.yaml

```diff
@@ -0,0 +1,38 @@
+{{- if and (not .Values.mariadb.enabled) .Values.api_db.initial_stamp -}}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "placement-stamp-alembic"
+  labels:
+    app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+    app.kubernetes.io/instance: {{ .Release.Name | quote }}
+    app.kubernetes.io/version: {{ default "unknown" .Chart.AppVersion }}
+    helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+  annotations:
+    "helm.sh/hook": pre-upgrade
+    "helm.sh/hook-delete-policy": before-hook-creation
+spec:
+  template:
+    metadata:
+      name: "placement-stamp-alembic"
+      labels:
+        app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+        app.kubernetes.io/instance: {{ .Release.Name | quote }}
+        helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    spec:
+      restartPolicy: Never
+      containers:
+      - name: stamp
+        image: {{ required ".Values.global.dockerHubMirrorAlternateRegion is missing" .Values.global.dockerHubMirrorAlternateRegion }}/{{ .Values.mariadb.image | default "library/mariadb:10.5.12" }}
```

Comment:
Why does this come from an the alternate region? Is that by choice or copy&paste like we had before?

Reply:
No, not by conscious choice, it is from the source.